libcontainer/process_linux.go

```diff
@@ -83,8 +83,7 @@ func (p *setnsProcess) start() (err error) {
 	if err = p.execSetns(); err != nil {
 		return newSystemErrorWithCause(err, "executing setns process")
 	}
-	// We can't join cgroups if we're in a rootless container.
-	if !p.config.Rootless && len(p.cgroupPaths) > 0 {
```

Comment:
I'm actually not sure what I meant either, sorry about that. This change is fine, but I would like to see a `runc exec` test with rootless cgroups in use, to make sure this works fine.